admin_manual/configuration_server/index.rst

```diff
@@ -22,5 +22,6 @@ ownCloud Server Configuration
    search_configuration   
    thirdparty_php_configuration
    performance_tuning
+   performance_tuning/index
```

Comment:
Nope, just had planned to split out the database and webserver part out of the performance_tuning.rst and leave the other parts there.